src/pymp4/parser.py

```diff
@@ -132,13 +132,11 @@ def _sizeof(self, context, path):
             "timescale" / Default(Int32ub, 10000000),
             "duration" / Int32ub,
         ),
-    })),
+    }),
     "rate" / Default(Int32sb, 65536),
     "volume" / Default(Int16sb, 256),
     # below could be just Padding(10) but why not
-    Const(Int16ub, 0),
-    Const(Int32ub, 0),
-    Const(Int32ub, 0),
+    Padding(10),
```

Comment:
Unnecessary change. It is more correct to go with the previous approach if its specific offsets/lengths are reserved.